dev/deployed-commit.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script determines the commit of sourcegraph/sourcegraph that is currently
+# running on sourcegraph.com. The returned version has one of the following
+# formats:
+#
+#   - v{semver}
+#   - {buildkite build num}_{date}_{abbreviated commit}
+#
+# In either case, the following cut will give us something parseable by rev-parse.
+
+git rev-parse "$(curl https://sourcegraph.com/__version | cut -d'_' -f3)"
```

Comment:
```suggestion
set -o pipefail
git rev-parse "$(curl -sf https://sourcegraph.com/__version | cut -d'_' -f3)"
```

I would add add `pipefail` and [-sf](https://explainshell.com/explain?cmd=curl+-sf)